data/xml/2021.ccl.xml

```diff
@@ -0,0 +1,1637 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2021.ccl">
+  <volume id="1" ingest-date="2021-08-31">
+    <meta>
+      <booktitle>"Proceedings of the 20th Chinese National Conference on Computational Linguistics"</booktitle>
+      <editor><first>Sheng</first><last>Li</last></editor>
+      <editor><first>Maosong</first><last>Sun</last></editor>
+      <editor id="yang-liu-ict"><first>Yang</first><last>Liu</last></editor>
+      <editor><first>Hua</first><last>Wu</last></editor>
+      <editor><first>Kang</first><last>Liu</last></editor>
+      <editor><first>Wanxiang</first><last>Che</last></editor>
+      <editor><first>Shizhu</first><last>He</last></editor>
+      <editor><first>Gaoqi</first><last>Rao</last></editor>
+      <editor><first>生</first><last>李</last></editor>
+      <editor><first>茂松</first><last>孙</last></editor>
+      <editor><first>洋</first><last>刘</last></editor>
+      <editor><first>华</first><last>吴</last></editor>
+      <editor><first>康</first><last>刘</last></editor>
+      <editor><first>万翔</first><last>车</last></editor>
+      <editor><first>世柱</first><last>何</last></editor>
+      <editor><first>高琦</first><last>饶</last></editor>
+      <publisher>Chinese Information Processing Society of China</publisher>
```

Comment:
None of this looks right. It looks like the Chinese names have been added as extra authors. They should be added as `<variant>` tags. See CCL 2020 for how it’s done.

Reply:
Okay, I'll look into this